.github/workflows/release-drafter.yml

```
@@ -0,0 +1,18 @@
+name: release-drafter
+
+on:
+  push:
+    # branches to consider in the event; optional, defaults to all
+    branches:
+      - master
+      - "releases/**"
+      - "stable/**"
+
+jobs:
+  update_release_draft:
+    runs-on: ubuntu-20.04
```

This is similar to how [prettytable has configured it](https://github.com/jazzband/prettytable/blob/5040d3b9e7d4f44cc901c8f44a5477fa57869632/.github/workflows/release-drafter.yml) and will restrict the permissions of the `GITHUB_TOKEN` a bit:

```suggestion
    permissions:
      contents: write
      pull-requests: read
    if: github.repository == 'jazzband/pip-tools'
    runs-on: ubuntu-latest
```